android-components/components/feature/search/src/test/java/mozilla/components/feature/search/telemetry/ads/AdsTelemetryTest.kt

```diff
@@ -109,7 +109,7 @@ class AdsTelemetryTest {
         assertEquals(Component.FEATURE_SEARCH, facts[0].component)
         assertEquals(Action.INTERACTION, facts[0].action)
         assertEquals(AdsTelemetry.SERP_ADD_CLICKED, facts[0].item)
-        assertEquals("google.in-content.organic.none", facts[0].value)
+        assertEquals("google.in-content.organic.other", facts[0].value)
```

Comment:
With these changes from `none` to `other`, there's something wrong here though I can't see what from reading the code.

If there is no partner code parameter, then we should be reporting `none`.

Reply:
I just realised we don't have good documentation on the intent of the probes, so I've just filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1848321) to hopefully improve that.

The last part of the key represents the code:

* ``<code>`` The parter code where known. This is typically only for partners associated with the product.
*  ``other`` The SERP load had a partner code, but it is not recognised as an associated partner or an organic code.
* ``none`` The SERP load had no partner codes, or it was a recognised organic code, e.g. some sites assign their own codes for searches.

So as there's no code at all, it should be `none`.